docs/admin/backup.rst

```diff
@@ -173,37 +172,36 @@ Database
 
 The actual storage location depends on your database setup.
 
-The database is the most important storage. Set up regular
-backups of your database, without it all your translation setup will be gone.
+The database is the most important thing to have regular backups of.
+Without it all the work going into translation is gone.
 
 Native database backup
 ++++++++++++++++++++++
 
-The recommended approach is to do dump of the database using database native
+The recommended way is to do save a dump of the database using database native
 tools such as :program:`pg_dump` or :program:`mysqldump`. It usually performs
-better than Django backup and restores complete tables with all data.
+better than Django backup, and it restores complete tables with all their data.
 
-You can restore this backup in newer Weblate release, it will perform any
+You can restore this backup in newer Weblate releases, where it performs all
 necessary migrations when running in :djadmin:`django:migrate`. Please consult
-:doc:`upgrade` on more detailed information how to perform upgrade between
-versions.
+:doc:`upgrade` on more detailed info on how to upgrade between versions.
 
 Django database backup
 ++++++++++++++++++++++
 
-Alternatively you can backup database using Django's :djadmin:`django:dumpdata`
+Alternatively, you can back up your database using Django's :djadmin:`django:dumpdata`
 command. That way the backup is database agnostic and can be used in case you
-want to change database backend.
+want to change the database back-end.
 
-Prior to restoring you need to be running exactly same Weblate version as was
-used when doing backups. This is necessary as the database structure does
-change between releases and you would end up corrupting the data in some way.
+Prior to restoring the database you need to be running exactly the same Weblate
+version the backup was made on. This is necessary as database structure changes
+between releases otherwise corrupts the data.
 After installing the same version, run all database migrations using
 :djadmin:`django:migrate`.
 
-Once this is done, some entries will be already created in the database and you
+Afterwards some entries will already be created in the database and you
```

Comment:
Will be duplicated? Is the backup changed?
I don't understand 202-203

Reply:
The database migration creates some entries in the database - for example permissions, roles, groups or users (but it's not limited to this).

Reply:
OK, resolved